forge-gui/res/cardsfolder/upcoming/legate_lanius_caesars_ace.txt

```diff
@@ -0,0 +1,13 @@
+Name:Legate Lanius, Caesar's Ace
+ManaCost:2 B R
+Types:Legendary Creature Human Soldier
+PT:2/2
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigRepeatEach | TriggerDescription$ Decimate — When CARDNAME enters the battlefield, each opponent sacrifices a tenth of the creatures they control, rounded up.
+SVar:TrigRepeatEach:DB$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ DBSacrifice
+SVar:DBSacrifice:DB$ Sacrifice | Amount$ X | SacValid$ Creature | Defined$ Remembered
+SVar:X:Count$Valid Creature.RememberedPlayerCtrl/TenthUp
```

Comment:
are you sure TenthUp exists?

Reply:
It did seem to work in testing but this is a pretty unique effect

Reply:
seems far from right lol:
![image](https://github.com/Card-Forge/forge/assets/8506892/f157e8ef-7f92-429e-8c1f-607f8df1ff09)
![image](https://github.com/Card-Forge/forge/assets/8506892/cd0cf815-e824-4782-9dca-a991aa28c898)

Reply:
Hmm yeah, should I manually calculate the 1/10 then ?

Reply:
if you've got a trick for that?

Reply:
Added `DivideEvenlyUp` for you in #4778